conftest.py

```diff
@@ -299,21 +299,49 @@ def strip_assertions_stderr(messages: Sequence[str]) -> list[str]:
     return res
 
 
-from pytest_pyodide.runner import NodeRunner
+from pytest_pyodide.runner import (
+    NodeRunner,
+    PlaywrightChromeRunner,
+    SeleniumChromeRunner,
+)
 
 
-def patched_load_pyodide(self):
+def patched_load_pyodide_node(self):
     self.run_js(
         """
         const {readFileSync} = require("fs");
         let snap = readFileSync("snapshot.bin");
         snap = new Uint8Array(snap.buffer);
-        let pyodide = await loadPyodide({ fullStdLib: false, jsglobals : self, _loadSnapshot: snap });
+        let pyodide = await loadPyodide({
+            fullStdLib: false,
+            jsglobals: self,
+            _loadSnapshot: snap,
+            enableRunUntilComplete: true,
+        });
+        self.pyodide = pyodide;
+        globalThis.pyodide = pyodide;
+        pyodide._api.inTestHoist = true; // improve some error messages for tests
+        """
+    )
+
+
+NodeRunner.load_pyodide = patched_load_pyodide_node
+
+
+def patched_load_pyodide_chrome(self):
```

Comment:
We'll need to move this patch and the jspi settings on lines 27 -- 33 upstream into `pytest-pyodide` to be able to use it in zarr. @agriyakhetarpal could you open a PR adding these changes there?

Reply:
I guess we need to implement some ways to make the runners configurable from downstream.